packages/shellharden/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/anordal/shellharden
+TERMUX_PKG_DESCRIPTION="The corrective bash syntax highlighter"
+TERMUX_PKG_VERSION=4.0
+TERMUX_PKG_SRCURL=https://github.com/anordal/shellharden/archive/v4.0.tar.gz
```

Comment:
`s/v4.0.tar.gz/v${TERMUX_PKG_VERSION}.tar.gz/g` perhaps?

Reply:
Thanks, I will fix that.